massgen/frontend/displays/timeline_event_recorder.py

```
@@ -131,8 +156,16 @@ def __init__(self, timeline: _MockTimeline, agent_id: Optional[str] = None) -> N
     def _get_timeline(self) -> _MockTimeline:
         return self._timeline
 
-    def start_new_round(self, round_number: int, is_context_reset: bool = False) -> None:
-        self._timeline.add_separator(f"Round {round_number}", round_number=round_number)
+    def start_new_round(
+        self,
+        round_number: int,
+        is_context_reset: bool = False,
```

Ruff (0.14.14)

[warning] Unused method argument: `is_context_reset`

(ARG002)